specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -12806,6 +15566,90 @@
         }
       }
     },
+    "ContinuousWebJob": {
+      "description": "Continuous Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ContinuousWebJob resource specific properties",
+          "properties": {
+            "status": {
+              "description": "Job status.",
+              "type": "string"
+            },
+            "detailedStatus": {
+              "description": "Detailed status.",
+              "type": "string"
+            },
+            "logUrl": {
+              "description": "Log URL.",
+              "type": "string"
+            },
+            "name": {
```

Comment:
Log URL?  It's the URL of the log file for the function (history of times the function was called, and any diagnostic information during each of the runs).
Name?  It's the function name, e.g. "MyFunction" or "DoSomething".